src/main/java/frc/robot/Telemetry.java

```diff
@@ -25,11 +26,14 @@ public class Telemetry {
    */
   public Telemetry(double maxSpeed) {
     MaxSpeed = maxSpeed;
+    SmartDashboard.putData(field);
   }
 
   /* What to publish over networktables for telemetry */
   NetworkTableInstance inst = NetworkTableInstance.getDefault();
 
+  Field2d field = new Field2d();
```

Comment:
Please use `final` for variables whenever possible. See [this Q&A from StackOverflow](https://stackoverflow.com/questions/27369584/what-does-the-jvm-do-with-a-final-variable-in-a-class) for one reason why.

```suggestion
  final Field2d field = new Field2d();
```